sql/src/main/antlr/OpenSearchSQLLexer.g4

```diff
@@ -327,6 +327,7 @@ TERMS:                              'TERMS';
 TOPHITS:                            'TOPHITS';
 TYPEOF:                             'TYPEOF';
 WEEK_OF_YEAR:                       'WEEK_OF_YEAR';
+WEEKOFYEAR:                       'WEEKOFYEAR';
```

Comment:
`'WEEKOFYEAR';` should be indented slightly more to be in line with all other functions

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/223/commits/26dcbc68710c800b69f0f8ea8daca01e25d0f803